spec/index.bs

```diff
@@ -2712,7 +2712,7 @@ A {{GPUBindGroup}} object has the following internal slots:
     ::
         The set of {{GPUBindGroupEntry}}s this {{GPUBindGroup}} describes.
 
-    : <dfn>\[[usedResources]]</dfn> of type map<[=subresource=], list<[=internal usage=]>>.
+    : <dfn>\[[usedResources]]</dfn> of type map<[=subresource=], [=list=]<[=internal usage=]>>.
```

Comment:
Would you mind taking a followup to change instances of `map` and `sequence` elsewhere in the spec to `[=ordered map=]` and `[=list=]`? As well as to escape any angle brackets that go with them (like here) (or you could decide to change the style to "[=list=] of [=internal usages=]" or something).